styles/main.scss

```diff
@@ -1470,7 +1484,7 @@ blockquote p:last-child {
   top: -43px;
 }
```

Comment:
@fordelord Hey, those one look like magic numbers as well, something to double check

Reply:
Hmm, with element which have position: absolute I think this can not be the magic number, but if we will remake it like transform: translate(-43px), will it be the magic number?🤔

Reply:
@1alexvash

Reply:
@1alexvash I read the lot of information about magic numbers in CSS. I fix magic numbers in the avatar and editor icon, but what about draft and future posts triangles. Must I refactor the triangles using only this numbers: 0, 25, 50 and HTML?) These triangles was implemented with a lot of magic numbers primordial three month ago.